Install.md

```diff
@@ -37,35 +31,70 @@ Starting with version 1.1.0, the GCM is capable of configuring Git automatically
     ```
     git-credential-manager install
     ```
+
     
-### RPM installation
+### Installing on Linux using RPM (recommended)
 
-1. Copy the `git-credential-manager-1.2.0-1.noarch.rpm` file somewhere locally.
-2. Install the RPM running as the root user:
+1. Download [git-credential-manager-1.2.0-1.noarch.rpm](https://github.com/Microsoft/Git-Credential-Manager-for-Mac-and-Linux/releases/download/git-credential-manager-1.2.0/git-credential-manager-1.2.0-1.noarch.rpm) and copy the file somewhere locally.
+2. Download the [PGP key used to sign the RPM](https://java.visualstudio.com/Content/RPM-GPG-KEY-olivida).
+3. Import the signing key into RPM's database:
 
     ```
-    sudo rpm -i git-credential-manager-1.2.0-1.noarch.rpm
+    sudo rpm --import RPM-GPG-KEY-olivida
+    ```
+4. Verify the GCM RPM:
+
+    ```
+    rpm --checksig --verbose git-credential-manager-1.2.0-1.noarch.rpm
+    ```
+    ...you should see a line (among those there) that is equal to the following:
+    
+    ```
+    V4 RSA/SHA256 Signature, key ID ba34dbc2: OK
     ```
-3. Run git-credential-manager in `install` mode, which will check its requirements and then update the "global" Git configuration file (the one in your home folder):
+5. Install the RPM:
+
+    ```
+    sudo rpm --install git-credential-manager-1.2.0-1.noarch.rpm
+    ```
+6. Run the GCM in `install` mode, which will check its requirements and then update the "global" Git configuration file (the one in your home folder):

```

Comment:
Line 29 is the same as this line but you changed from git-credential-manager to GCM here. Could you change that line to match?